### PR TITLE
Corr&opt version overhaul

### PR DESCRIPTION
**Pull request by Th3M3**
**By submitting this pull request, I confirm the following (please check boxes, eg [X] - no spaces) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

_3_

---

Some optimizations to #517
@DL6ER plz have a look into it and let me know if hiding the version infos in footer and dropdown-menu  depending to screen width work now as expected

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

File: scripts/pi-hole/php/footer.php [hunk 1]
```diff
@@ -41,32 +41,21 @@
...
-	}
-	a.lookatme {
-		-webkit-animation-name: Pulse;
-		-webkit-animation-duration: 2s;
-		-webkit-animation-iteration-count: infinite;
-	}
-</style>
-        <div class="pull-right hidden-xs <?php if(isset($core_commit) || isset($web_commit)) { ?>hidden-md<?php } ?>">
+	<!-- Version Infos -->
+        <div class="pull-right hidden-xs <?php if(isset($core_commit) || isset($web_commit)) { ?>hidden-sm<?php } ?>">
```
Comment (DL6ER) [inline scripts/pi-hole/php/footer.php]: Suggestion: `<div class="pull-right hidden-xs hidden-sm <?php if(isset($core_commit) || isset($web_commit)) { ?>hidden-md<?php } ?>">`

File: scripts/pi-hole/php/header.php [hunk 2]
```diff
@@ -266,23 +266,23 @@ function pidofFTL()
                             </li>
                             <!-- Menu Footer -->
                             <li class="user-footer">
-                                <!-- Update alerts -->
-                                <div>
+                                <!-- Version Infos -->
+                                <div class="<?php if(!isset($core_commit) && !isset($web_commit)) { ?>hidden-md <?php } ?>hidden-md hidden-lg">
```
Comment (DL6ER) [inline scripts/pi-hole/php/header.php]: See above.

Comment (DL6ER) [inline scripts/pi-hole/php/header.php]: Suggestion: `<div class="<?php if(!isset($core_commit) && !isset($web_commit)) { ?>hidden-md<?php } ?> hidden-lg">`

File: scripts/pi-hole/php/footer.php (hunk 1 above)
Comment (Th3M3) [inline scripts/pi-hole/php/footer.php]: I also thouht about that after I saw your tablet landscape screenshot.
Gladly you can change it to your suggestion, but I would move the space before the by-php-added class into the php code, like this:
`<div class="pull-right hidden-xs hidden-sm<?php if(isset($core_commit) || isset($web_commit)) { ?> hidden-md<?php } ?>">`

File: scripts/pi-hole/php/header.php (hunk 2 above)
Comment (Th3M3) [inline scripts/pi-hole/php/header.php]: here too,
do the space after the by-php-added class into the php-code:
`<div class="<?php if(!isset($core_commit) && !isset($web_commit)) { ?>hidden-md <?php } ?>hidden-lg">`

File: scripts/pi-hole/php/footer.php (hunk 1 above)
Comment (DL6ER) [inline scripts/pi-hole/php/footer.php]: I'm okay with that. I'll approve your changes once you push them. Same for the occurrence in `header.php`.

**Comment by Th3M3** [comment]
@DL6ER OK, changes done.

**Comment by PromoFaux** [comment]
Approved

**Pull request merged**